core/typescript/src/lib/bootstrap/cdk8s/chart.ts

```diff
@@ -0,0 +1,91 @@
+import { App, Chart, ChartProps } from 'cdk8s';
+import { Construct } from 'constructs';
+
+import {
+  ComponentConfig,
+  GeneralConfig,
+  InfraConfig,
+  NetworkConfig,
+  RelationConfig,
+  ServiceConfig,
+} from '../../config';
+import { Orchestrator } from '../../orchestrator';
+import { InfraPlanConstructs, Planner, ResultHandler } from '../../package';
+import { Custom } from '../../project';
+
+/**
+ * Properties for {@link ProjectChart}.
+ *
+ * @group For project creators
+ */
+export interface ProjectChartProps<
```

Comment:
I've not spent much time thinking about K8s and not looked closely at CDK8s. Given the word "charts" presume it operates at the service/app level?

If so this only applies to a CDK `Service` type construct. Never a component. does Network and Relation ever apply?

Reply:
Being stand alone probably makes sense though. like each microservice repo would have a CDK8s project. Maybe in a macroservice type setup relations would make sense like [Uber domain-oriented microservice architecture](https://www.uber.com/blog/microservice-architecture/). But that's a pretty niche use case I think.

Reply:
Good point. I think right now it is probably how it works, i.e. one cdk8s chart => one k8s chart.

Reply:
> Left some comments.
> 
> Icons - don't see them referenced anywhere, assume will be used in docs to come?

When we do the release I will manually update that release PR to include those icons